### ISSUE TITLE
🐛 Most white mages should not cast holy

### ISSUE THREAD

**Issue by WinterSolstice8** [DerpyProjectGroup/topaz#5484]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## OS / platform the server is running (if known)

All

## Branch affected by issue 

`base`

## Steps to reproduce

All generic white mages cast holy

## Expected behavior

Not all white mages should cast holy

This is a non-exhaustive list from old logs around 2006-2008

```
The Kindred White Mage casts Holy
The Vanguard Alchemist casts Holy
The Mamool Ja Blusterer casts Holy
The Yagudo Abbot casts Holy
The Ul'aern casts Holy
The Ru'aern casts Holy
The Divine Sentinel casts Holy
The Goblin Corpsman casts Holy
The Ace of Cups casts Holy
The Imperial Trooper casts Holy
The Yagudo Abbot casts Holy
The Imperial Trooper casts Holy
The Olla Grande casts Holy
The Vanguard Amputator casts Holy
The Imperial Trooper casts Holy
The Ace of Cups casts Holy
The Yagudo Priest casts Holy
The Mamool Ja Executioner casts Holy
The Imperial Trooper casts Holy
The Vanguard Alchemist casts Holy
```
and Di'Dha Adamantfist

tl;dr: holy is a more exclusive spell, limited to NMs and high level yagudo and a few others

**Comment by gweivyth** [comment]
Hi Winter,

Do you have any idea as to what the best way to get a more exhaustive (and possibly up to date) list would be?  I was thinking maybe pulling an entire zone and parsing a chat log while afk on WHM with Regen rolling, but I am open to better ideas.  Let me know and I can get more data.  I have excess free time during the day now for gathering this sort of data.
